src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,65 @@
+function currencyNameHandler (metric, options) {
```

Comment:
This function is not readable at all - I cannot check it.
Please add documentation what it does and we'll figure out a better way to do it.

Reply:
You're right, I was able to figure out a way for it to look better, I don't think it needs a documentation now